### PR TITLE
Added calculator component

### PR DESCRIPTION
**Pull request by Rana-Mudassir**
**Developed a React component**
- Created Calculator Component
- Add styling to Calculator Component
- Used React function Component

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+function Calculator() {
```
Comment (1GORDON) [inline src/components/Calculator.js:4]: Great job @Rana-Mudassir 🤝

- [x] As stated in the project requirements, kindly remove all the boilerplate that got created by the CRA (texts, images, and styles). You may delete these files `logo.svg`, `App.test.js`, `reportWebVitals.js`, `setupTests.js`.
- [ ] Kindly ensure `Calculator.js` should have at least 1 child component.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
```
Comment (1GORDON) [inline src/App.js]: Great job @Rana-Mudassir 🤝

- [x] You may always remove commented code from your project as this is regarded as a bad practice.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Rana-Mudassir ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @Rana-Mudassir,

## APPROVED 🟢

## Highlights
✔️ Child component for calculator component

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**